src/backend/main.ts

```diff
@@ -159,19 +147,39 @@ import {
   getMainWindow,
   sendFrontendMessage
 } from './main_window'
-import {
-  addGameToLibrary,
-  getHyperPlayGameInfo,
-  getHyperPlayGameInstallInfo,
-  uninstallHyperPlayGame
-} from './hyperplay/library'
+import { addGameToLibrary } from './storeManagers/hyperplay/library'
 
-import {
-  importGame,
-  isHpGameAvailable,
-  isHpGameNative,
-  stopHpGame
-} from './hyperplay/games'
+import * as HyperPlayGameManager from './storeManagers/hyperplay/games'
```

Comment:
Would it be an idea to move the map composition to ./storeManagers/index.ts, in order to de-clutter main.ts a wee bit?

Reply:
good idea. updated now